README.md

```diff
@@ -111,33 +111,20 @@ Then, Select the Python binary as `venv` in VSCode:
 
```

Comment:
Can you please update the README to reflect the upgrade to python 3?   Currently it has instructions to load 2.7.16 via `pyenv`

Reply:
Agreed I will change it